### PR TITLE
Add a bunch of translations to Hebrew

### PR DESCRIPTION
**Pull request by Don-Vito**
A sample bunch. 
This job is much worse than I thought 😆

### CODE REVIEW

File: Files/MultilingualResources/Files.he-IL.xlf [hunk 1]
```diff
@@ -52,7 +52,7 @@
         </trans-unit>
         <trans-unit id="NavigationToolbarCopyPath.Text" translate="yes" xml:space="preserve">
           <source>Copy location</source>
-          <target state="translated" state-qualifier="tm-suggestion">העתק נתיב</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">העתק מיקום</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.he-IL.xlf:55]: All translated strings should be marked as translated.

Comment (Don-Vito) [reply Files/MultilingualResources/Files.he-IL.xlf:55]: Not sure I follow. Are you addressing this specific line or in general? 

Here I took something that was translated inconsistently and changed the translation -  how should it be tagged?

Comment (XPoppyX) [reply Files/MultilingualResources/Files.he-IL.xlf:55]: @Don-Vito I created a PR in your repository, take a look

Comment (yaichenbaum) [reply Files/MultilingualResources/Files.he-IL.xlf:55]: @Don-Vito Thank you for opening this pull request 😂
tsvietOK is requesting that you change state of translated strings from `needs-review-translation` to `translated`.

Comment (Don-Vito) [reply Files/MultilingualResources/Files.he-IL.xlf]: @XPoppyX  - thanks for the PR!
@yair100 - is there a description for the state-machine somewhere? Is it always switching from `new` to `translated`? We could probably add this as a Contribution section in README.

Comment (yaichenbaum) [reply Files/MultilingualResources/Files.he-IL.xlf]: @Don-Vito Most people do the translations through the Multilingual App Toolkit Editor which handles the state automatically, however we can definitely add some information about doing it manually.

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@Don-Vito Thank you! LGTM

**Pull request merged**